lib/MusicBrainz/Server/Controller/Root.pm

```diff
@@ -166,7 +166,10 @@ sub error_404 : Private {
 
     $c->response->status(404);
     $c->stash->{current_view} = 'Node';
-    $c->stash->{component_props}{message} = $message;
+    $c->stash->{component_props} = {
+        message => $message,
+        template => 'main/error/404',
```

Comment:
This change seems unrelated to the commit and possibly a no-op (re the `template` addition), unless I'm missing something.

Reply:
I'm moving the file from main/404 to main/error/404, wouldn't that require specifying the template?

Reply:
I was being an idiot, fixed that now.